course3/meeting2/hello.cc

```diff
@@ -1,13 +1,56 @@
 #include <iostream>
-#include <string>
 
+class SimpleString
+{
+  public:
+    SimpleString()
+    {
+    }
+
+    SimpleString(const char* ascii)
+    {
+      int lastCharacterIndex = BUFFER_SIZE - 1;
+
+      int i = 0;
+      while(i < lastCharacterIndex
```

Comment:
Because 'for' is canonical for of this kind of loop. More obvious to the reader;. Still, std::strcpy is _the_ cleanest solution.